recipe/meta.yaml

```diff
@@ -6,23 +6,23 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/s2downloader-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/s2downloader-{{ version }}.tar.gz
   sha256: df646606f3d32a8cafa26510b9d55e5db2ccbaa084342d0e2f8177ab70e1e1a8
 
 build:
   entry_points:
     - s2downloader=s2downloader.s2downloader_cli:main
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 0
+  number: 1
 
 requirements:
   host:
-    - python >=3.7
```

Comment:
@DaRa42 You can override the default minimal Python version here. Just replace `python {{ python_min }}` with `python >=3.10` in the host and the run section of the meta.yaml.